src/js/edu-benefits/1990e/config/form.js

```diff
@@ -117,18 +116,20 @@ const formConfig = {
                 }
               }
             },
-            veteranSocialSecurityNumber: _.set(['ui:title'], 'Veteran Social Security number', ssnUiSchema),
+            // There must be a more succinct way for lodash to set 2 properties
+            veteranId: _.set(['vaFileNumber', 'ui:title'], 'Veteran File number',
+                             _.set(['veteranSocialSecurityNumber', 'ui:title'], 'Veteran Social Security number', veteranId.uiSchema)),
             veteranAddress: address.uiSchema('Veteran Address'),
             serviceBranch: {
               'ui:title': 'Branch of Service'
             }
           },
           schema: {
             type: 'object',
-            required: ['veteranFullName', 'veteranSocialSecurityNumber'],
+            required: ['veteranFullName'],
             properties: {
               veteranFullName: fullName,
-              veteranSocialSecurityNumber: ssn,
+              veteranId: veteranId.schema,
```

Comment:
This should be `view:veteranId`, I think, so that we don't send a veteranId object to the backend.